### DISCUSSION TITLE
A quiet sound in osu!lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#13372] [Q&A] [answered]
Recently, I started to run osu! Lazer very often, I love its interface, and the pen in osu! Lazer does not feel any input lag at all, so I have a lot of fun in it. However, I noticed one strange thing, the sound in the osu! Lazer is slightly quieter than in the stable version, and this causes a little discomfort when playing. At first I thought the problem was with my computer or headphones, but one day in the osu! on a popular social network in Russia, I saw a [post](https://vk.com/osu.content?from=quick_search&w=wall-101391911_40589), which spoke of this as a fact, so the problem is not only mine. Is it possible to make the sound as loud as it was in the stable version? Was it planned to make the sound in the osu! Lazer quieter than in the stable version, and if so, what was it for?

**Comment by peppy** [comment, marked as answer]
This was done recently to allow for more head-room in the interface samples (we're tracking this at https://github.com/ppy/osu/issues/12646).

In the next release we plan to boost the music volume back up (see https://github.com/ppy/osu/pull/13328) until we fully implement the normalisation system (which is coming in the next few months to fix this in a better way).
